api/src/index.js

```diff
@@ -73,6 +73,7 @@ app.use("/diagoriente", require("./controllers/diagoriente"));
 app.use("/bus", require("./controllers/bus"));
 app.use("/support-center", require("./controllers/support-center"));
 app.use("/support-center/knowledge-base", require("./controllers/knowledgeBase"));
+app.use("/", require("./controllers/signin"));
```

Comment:
C'est plus cohérent si le controlleur `signin` est derrière une route `signin` non ?

```suggestion
app.use("/signin", require("./controllers/signin"));
```